### ISSUE TITLE
Unable to use UIkit in TypeScript

### ISSUE THREAD

**Issue by gokkep** [uikit/uikit#3084]
### UIkit version

3.0.0-beta.35

### Steps to reproduce
Setup a Visual Studio Code environment, install npm, install TypeScript transpiler (v2.5.3), install WebPack (v3.8.1), install UiKit (v3.0.0 beta 35)

Use the following code in the entry file indicated by the webpack.config.js

import UIkit from 'uikit'
import Icons from 'uikit/dist/js/uikit-icons'

UIkit.use(Icons)
UIkit.notification('Hello world.')

### What is Expected?
Code that runs,

### What is actually happening?
Using the debugger of Visual Studio code which runs an instance of Google Chrome in debug mode will stop at the line UIkit.use(Icons) with the following error:

Exception has occurred: TypeError
TypeError: Cannot read property 'use' of undefined

So the UIkit property is undefined.

### Additional information
When I use the same code in a javascript (.js) entry file it works!

**Comment by dasdeck** [comment]
My typescript is kinda basic, but as I understand we'd either need someone to create typings for the current version of UIkit or, In the meantime, you should be able to get by using the `--allowJs` compiler flag. https://www.typescriptlang.org/docs/handbook/compiler-options.html

Cheers!

**Comment by dasdeck** [comment]
@gokkep 
I created a minimal example on how to bundle UIkit inside a TypeScript environment. 
https://github.com/dasdeck/uikit-example

Please let me now if that helps!

Cheers!

**Comment by gokkep** [comment]
Hi there,

Finally found some time to look at this.

Tried your example, worked and gave me the insights I needed, also learned how to make more use of npm in relation with installing and building TypeScript projects in combination with webpack.

For me the solution seems to be the suggestion to use `import * as UIkit`, instead of `import UIkit from`.

I did not have to use your first suggestion of using the `--allowJs` compiler flag.

So for now I am satisfied, although I do not understand the difference in the import statements, I will have to dig into that, to understand it and if it has consequences.

First I will try to use UIkit the coming period of time and if I am happy with it, I will look into adding the typings file for TypeScript in combination with UIkit, although I personally think it would be better that it is included in the UIkit issue list for the UIkit developers to expand the possibilities of their framework. I personally think TypeScript is a good environment for developers like me coming from the Object Oriented world and not from a JavaScript background and making it possible to write robust, exapndable and reusable software components for user agents like the browsers.

For now thank you very much for the support!

Greetz, Paul.
